prost-types/src/lib.rs

```diff
@@ -23,13 +31,15 @@ pub mod compiler {
 // because the Protobuf versions are signed. To make them easier to work with, `From` conversions
 // are defined in both directions.
 
+#[cfg(feature = "std")]
```

Comment:
I think it'd be cleaner to move this code to a submodule, either inline or in a new file.  Then that whole module can be optionally compiled with a single `cfg` tag.

Reply:
done